Adaptors/MongoDB/src/TaskTable.cs

```diff
@@ -588,9 +588,9 @@ await foreach (var taskId in taskCollection.AsQueryable(sessionHandle)
                                                                                                           TaskStatus.Dispatched,
                                                                                                         }));
 
-    Logger.LogDebug("Release task {task} on {podName}",
-                    taskId,
-                    ownerPodId);
+    Logger.LogInformation("Release task {task} on {podName}",
```

Comment:
Isn't `task` already in the log scope ?
BTW, maybe it'd be a good Idea to have `podname` in a wider scope.

Reply:
TaskId should be in the scope. I added the podId in the scope. But further rework on logs is out of scope. We need to improve our logs but it is a issue in itself.